src/display/annotation_layer.js

```diff
@@ -858,14 +846,37 @@ class TextWidgetAnnotationElement extends WidgetAnnotationElement {
    *
    * @private
    * @param {HTMLDivElement} element
-   * @param {Object} font
    * @memberof TextWidgetAnnotationElement
    */
-  _setTextStyle(element, font) {
-    // TODO: This duplicates some of the logic in CanvasGraphics.setFont().
+  _setTextStyle(element) {
+    const TEXT_ALIGNMENT = ["left", "center", "right"];
+    const { fontName, fontSize, fontColor } = this.data.defaultAppearanceData;
     const style = element.style;
-    style.fontSize = `${this.data.fontSize}px`;
-    style.direction = this.data.fontDirection < 0 ? "rtl" : "ltr";
+
+    /**
+     *  TODO: If font size is 0, calculate the font-size based on the height and
+     *          width of the element
+     *  For now, just use the default font size
+     */
+    if (fontSize) {
+      style.fontSize = `${fontSize}px`;
+    }
+
+    style.color = `rgb(${fontColor[0]}, ${fontColor[1]}, ${fontColor[2]})`;
```

Comment:
You must use `Util.makeHexColor` rather than manually doing this.

Also, the commit message should be updated to mention that you also set the colour now.